olive/cli/generate_adapter.py

```diff
@@ -39,6 +39,13 @@ def register_subcommand(parser: ArgumentParser):
             choices=["float16", "float32"],
             help="The precision of the optimized model and adapters.",
         )
+        sub_parser.add_argument(
+            "--adapter_format",
+            type=str,
+            default=WeightsFileFormat.NUMPY,
+            choices=[el.value for el in WeightsFileFormat],
```

Comment:
We want to use ONNX_ADAPTER is the default value.

Reply:
done